dev/fattest.simplicity/bnd.bnd

```diff
@@ -60,6 +61,10 @@ Private-Package: \
 	com.ibm.websphere.simplicity.configuration
 
 Include-Resource: \
+    @${repo;org.bouncycastle:bcpg-jdk15on;1.69;EXACT},\
+    @${repo;org.bouncycastle:bcpkix-jdk15on;1.69;EXACT},\
+    @${repo;org.bouncycastle:bcprov-jdk15on;1.69;EXACT},\
+    @${repo;org.bouncycastle:bcutil-jdk15on;1.69;EXACT},\
```

Comment:
Are you sure that this is the best version to use?
If you look here - https://www.bouncycastle.org/latest_releases.html - you will find that there is a 1.73 release and that instead of using `jdk15on`, it seems to suggest using `jdk18on`.
> Java Version Details With the arrival of Java 15. jdk15 is not quite as unambiguous as it was. The jdk18on jars are compiled to work with anything from Java 1.8 up. They are also multi-release jars so do support some features that were introduced in Java 9, Java 11, and Java 15. If you have issues with multi-release jars see the jdk15to18 release jars below.